File: src/main/kotlin/a/Presents.kt
package a

/* Problem Link: https://codeforces.com/problemset/problem/136/A */

fun main() {
    val n = readln().toInt()
    val friends = readln().split(" ").map { it.toInt() }

    val sortedList = List(friends.size) { index ->
        friends.indexOf(index + 1) + 1
    }

    for(x in sortedList) print("$x ")
}

File: Codeforces/C++/problemset/Presents.cpp
// Problem Link: https://codeforces.com/problemset/problem/136/A

#include <iostream>
using namespace std;
typedef long long ll;
int main()
{
    ll n, b;
    cin >> n;
    ll *p = new ll[n];
    for(ll i = 0; i < n; i++)
    {
        cin >> b;
        p[b-1] = i+1;
    }
    for(ll i = 0; i < n; i++)
        cout << p[i] << " ";
    delete [] p;
    return 0;
}
